### ISSUE TITLE
Continue #572: Problem with BBCode and Markdown

### ISSUE THREAD

**Issue by NightYoshi370** [erusev/parsedown#573]
Yeah, I realised. Just that my board members love BBCode more than Markdown, but we like having markdown on the wiki.

Also, uh, another thing I've realised:

`Posted by "DestroyerXL44" id="3297"`

But the &-quot; thing replaces the " (sorry, github formatting)

I've tried `->setMarkupEscaped(false),` but it didn't fix that.

I've also tried `setSafeMode(true),` but neither did that. fix it

For reference, below is my configuration:

```php
	$text = Parsedown::instance()
		->setUrlsLinked(false)		# prevents automatic linking of URLs
		->setMarkupEscaped(false)
		->line($text);
```

**Comment by NightYoshi370** [comment]
@aidantwoods I made a new issue, like you asked (in case you didn't notice)

**Comment by aidantwoods** [comment]
Thanks for reporting this @NightYoshi370.

Double quotes should always be permitted in the body of a HTML element that Parsedown produces – we only need to encode quotes when they go into an attribute (e.g. like when setting the `alt` description of an image). This to say that your example text should work regardless of escaping settings :)

There is a fix in #574. It ended up being a bit of a bigger change than I thought it would be initially, but encompassed some changes I was meaning to do anyway after #569.
